drawBot/context/baseContext.py

```diff
@@ -1677,7 +1677,9 @@ class BaseContext(object):
     _gradientClass = Gradient
 
     fileExtensions = []
-    saveImageOptions = []
+    saveImageOptions = [
+        ("multipage", None),
+    ]
```

Comment:
I left that list empty because I only want `multipage` to be documented for context that actually do something with it. And why would the doc string be None?